_includes/saved_searches.md

```diff
@@ -45,18 +45,18 @@ To delete a saved search from your list:
   </tr>
 </table>
 
-### Set A Saved Search as the Default Search
+### Set a Saved Search as the Default Search
 
 You can manage your saved searches to set a saved search as your default search.
 
-User's with the **Applications** permission can select a saved search as an Organization Default and set a saved search as the default search for all users. If you want to update the default search result, you need to save a new search and follow the steps below to make it your default search.
+Users with the **Applications** permission can select a saved search as an Organization Default and set a saved search as the default search for all users. If you want to update the default search result, you need to save a new search and follow the steps below to make it your default search.
 
 1. Click the <strong>Lists</strong> button to the left of the query.
     ![List button](/images/saved_searches_list_button.png)
 1. Click **Manage**.
 1. Set the default search:
-    * If you are a user with **Applications** permissions, you can set a default search for all users. Click the drop-down under the **Set Default** column, and select **Organization Default**.
-    * If you want to update the default saved search or set a new default saved search, select the saved search, click the drop-down under the **Set Default** column, and select **My Default**.
+    * If you are a user with **Applications** permission, you can set a default search for all users. Click the drop-down under the **Set Default** column, and select **Organization Default**.
```

Comment:
Click the drop-down menu under the **Set Default** column, and select **Organization Default**.